enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -247,7 +250,21 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
     } catch (ComponentLookupException e) {
       throw new EnforcerRuleException("Unable to lookup a component " + e.getMessage(), e);
     } catch (DependencyResolutionException e) {
-
+      Throwable cause = e.getCause();
+      while (cause != null) {
```

Comment:
This would be slightly more clear as a for loop:

```
for (Throwable cause = e.getCause(); cause != null; cause = cause.getCause())
```

This lets you drop the else clause